console/console-init/ui/src/modules/address-space/components/AddressSpaceToolbar/AddressSpaceToolbar.tsx

```diff
@@ -65,13 +65,19 @@ export const AddressSpaceToolbar: React.FunctionComponent<IAddressSpaceToolbarPr
     });
   };
 
+  const onSelectKebabDeleteAll = async (event: any) => {
+    if (event.target.value === "deleteAll") {
```

Comment:
What happens when the value is 'Purge All'?

Reply:
@anukritijha , There is no purgeAll option in address space list.

Reply:
This check already there. I don't why, i just move the code and as concern about other value as "PurgeAll" , we have only delete all action.